tools/ctor_evaller.py

```diff
@@ -125,7 +125,7 @@ def add_func(asm, func):
 var stackMax = stackTop + totalStack;
 if (stackMax >= totalMemory) throw 'not enough room for stack';
 
-var dynamicTop = stackMax;
+var dynamicTopPtr = stackMax;
```

Comment:
and a simple string replace can do that. this is before minification, so just find `function _sbrk(` and e.g. change that to `function _sbrk_should_abort(`.

also remove sbrk from the other places in `ctor_evaller.py` that assumed it was imported